persistence/sql/sql_process_store_common.go

```diff
@@ -78,3 +79,69 @@ func insertImmediateTask(
 
 	return tx.InsertImmediateTask(ctx, immediateTaskRow)
 }
+
+func (p sqlProcessStoreImpl) publishToLocalQueue(
+	ctx context.Context, tx extensions.SQLTransaction, processExecutionId uuid.UUID, messages []xdbapi.LocalQueueMessage) error {
+	for _, message := range messages {
+		dedupId := uuid.ParseUUID(message.GetDedupId())
+		if dedupId == nil {
```

Comment:
It's is very rare, but rather than silently swallow the error and use a random UUID, it would be better to just return an error.

Reply:
Done.